####-repo-rename.md

```diff
@@ -0,0 +1,150 @@
+    ibmq-docs-out-of-metapackage    :ibmq-docs, after rfc-merged, 2w
+    docs-out-of-metapackage         :docs, after ibmq-docs, 4w
+    benchmarks-out-of-metapackage   :benchmarks, after docs, 4w
+    tutorials-out-of-metapackage    :tutorials, after benchmarks, 4w
+    section archive-metapackage
+    ibmq-out-of-metapackage        :ibmq-archived, after release025, 1d
+    move-setup-out-of-metapackage  :setup, after release025, 1d
+    pre-archive-doc                :archive-doc, after release025, 2d
+    archive-metapackage            :archive-meta, after archive-doc setup, 1d  
+    section final
+    final-rename    : finalrename, after archive-meta, 1d
+    fix-time:       : fix-time, after finalrename, 2w
+    final-announcement : milestone, finalannoun
+```
+
+### preparation stage
+
+This stage covers the pre-work with the following independent tasks:
+
+ * **metapackage-repo**: Renaming `qiskit/qiskit` to `qiskit/qiskit-metapackage` is independently desired, as the new name describes more correctly its content.
+ * **rfc-discussion**: During 10 days after the submission of this RFC, this document will collect comments and feedback on the best way to mitigate the effects of this renaming.
+
+### empty-metapackage stage
+
+The main stage of the plan is to empty the metapackage to minimize the traffic to `qiskit/qiskit` (at this point, `qiskit/qiskit-metapackage`) before the renaming `qiskit/qiskit-terra` to `qiskit-qiskit`.
+
+For each task in this stage, the code is moved with related issues and PRs.
+The tasks are independent and can start at any point, but they need to finish before the qiskit-terra release of July.
+
+* **ibmq-docs-out-of-metapackage**: Early in the stage, we can try to deploy `qiskit-ibmq-provider` documentation as stand-alone in `qiskit.org/ecosystem/qiskit-ibmq-provider`. This is optional, as the full documentation will be removed on July.
+* **docs-out-of-metapackage**: Move documents from `qiskit/qiskit-metapackage` to `qiskit/qiskit-terra`. All the ["general documents"](https://github.com/Qiskit/qiskit/tree/master/docs) in needs to be moved to the `qiskit/qiskit-terra` to be part of a single sphinx build process. The [`qiskit-ibmq-provider` documentation](https://qiskit.org/documentation/apidoc/ibmq-provider.html) cannot be move to `qiskit/qiskit-terra`. Therefore, the options are:
+	* Wait until the stage ibmq-out-of-metapackage when the documentation can be removed as the repo `qiskit/qiskit-ibmq-provider` gets archived.
+	* Create a landing page in `qiskit.org/documentation/ibmq-provider` or `qiskit.org/ecosystem/ibmq-provider` [like Aer did](https://github.com/Qiskit/qiskit-aer/pull/1589).  
+* **benchmarks-out-of-metapackage**: Move benchmarks from `qiskit/qiskit-metapackage` to `qiskit/qiskit-benchmarks` (repo to create). Change the related scripts to continue running and deploying <https://qiskit.github.io/qiskit/>.
```

Comment:
Two things on this item. We don't necessarily have to do it via a separate repository we could put the benchmarks in the qiskit-terra (future qiskit) repo. We also don't have to use github pages for hosting either. It's just generating a static webpage so if we wanted we can put that on `qiskit.org/benchmarks` or something like that too. Both can be decided independently, basically we have 4 options:

1. New repo & github pages
2. qiskit-terra repo & github pages
3. New repo & qiskit.org hosting
4. qiskit-terra repo and & qiskit.org hosting

If we were determined to keep https://qiskit.github.io/qiskit/ then the only option to do this is option 2. Historically I've been opposed to option 2 because it led to a lot of repository bloat to contain the gh pages branch in the repo. But a recent configuration change to force push a single commit instead of maintaining the change history reduces that a lot. 

Personally, I don't think having continuity for the existing benchmark results is super critical. As long as all the data is there just using a new url isn't a big deal, so we should pick the option that we like best.

Reply:
Options added in f1a7289ac710696de76b474031df13b8912e9252 . I also do not feel very strong about any option in particular, but I lean towards not having them in qiskit.org, as it is a dev tool (it can be linked in the contributor guidelines).